$NetBSD: patch-ab,v 1.12 2023/04/30 14:58:58 spz Exp $

raise data and file descriptors limits to their hard limits

--- scripts/news.daily.in.orig	2014-05-17 08:24:49.000000000 +0000
+++ scripts/news.daily.in
@@ -276,6 +276,10 @@ fi
 
 cd ${PATHETC}
 
+## Unlimit
+ulimit -d `ulimit -H -d`
+ulimit -n `ulimit -H -n`
+
 ##  Show the status of the news system.
 ${DOSTAT} && {
     ${INNSTAT}
